man/systemd.link.xml

```diff
@@ -1062,6 +1062,51 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[VirtualWLANInterface] Section Options</title>
```

Comment:
Hmm, we usually don't suffix the sections in .link files with "Interface" (and not with "Link" either), given they are already located in per-*link* files anyway it's hence redundant.

More importantly though: this is weird in .link? We usually create neteowkr devices via .netdev files, not via .link. Hence we depart from that logic? i.e. .link is for configuring device properties, .netdev is for *creating* net devices, and .network for configuring network properties. This one appears clearly a case for .netdev to me

Reply:
Oh. That makes sense for me. I will move this to .netdev. Thanks.